packages/console/src/pages/ApplicationDetails/components/DeleteForm/index.tsx

```diff
@@ -24,9 +24,16 @@ const DeleteForm = ({ id, name, onClose }: Props) => {
 
   const [inputName, setInputName] = useState('');
   const [loading, setLoading] = useState(false);
+  const deleteButtonRef = useRef<HTMLButtonElement>(null);
 
   const inputMismatched = inputName !== name;
 
+  useEffect(() => {
+    if (!inputMismatched) {
+      deleteButtonRef.current?.focus();
+    }
+  }, [inputMismatched]);
+
```

Comment:
i don't think this is needed since it's by design to confirm deletion carefully

Reply:
But it doesn't mean we don't need keyboard support. Force input of the deleting resource name is enough, IMO

Reply:
And focusing on the button does not mean to trigger the deletion automatically either. User will still need to press enter to trigger delete action.
Yesterday I tried to delete some testing applications and found the current user experience is way too cumbersome. So I came up with this PR

Reply:
i think auto focus on input is good. but not the button

Reply:
you can confirm with guamian about the button behavior

Reply:
OK, will check with her.

Reply:
Removed for now